sdl-app/WorldObjectRenderer.cpp

```diff
@@ -26,19 +31,72 @@ namespace {
 void WorldObjectRenderer::Visit(const Ant& visitee) const {
     static constexpr auto kAntSize = 0.5;
 
-    SDL_SetRenderDrawColor(m_renderer, 0xFF, 0x00, 0x00, 0xFF);
+    SDL_SetRenderDrawColor(m_game->m_renderer, 0x94, 0x58, 0x0A, 0xFF);
 
     auto expanded = ExpandPointToRect(visitee.GetPosition(), kAntSize);
-    auto rect = BoxToFRectTransform(expanded, *m_camera);
-    SDL_RenderFillRectF(m_renderer, &rect);
+    auto rect = BoxToFRectTransform(expanded, m_game->m_camera);
+    SDL_RenderFillRectF(m_game->m_renderer, &rect);
 }
 
 void WorldObjectRenderer::Visit(const Plant& visitee) const {
     static constexpr auto kFoodSize = 0.2;
 
-    SDL_SetRenderDrawColor(m_renderer, 0x00, 0xFF, 0x00, 0xFF);
+    if(m_game->m_debugMode)
+    {
+        //this opens a font style and sets a size
+        TTF_Font* Sans = TTF_OpenFont("D:/Ant/AntGame/assets/cour.ttf", 48);
```

Comment:
need to go over all of this font stuff, e.g. add ScopeGuard for managing some off these font objects